utilities/write_batch_with_index/write_batch_with_index.cc

```diff
@@ -555,6 +565,15 @@ void WriteBatchWithIndex::MultiGetFromBatchAndDB(
     DB* db, const ReadOptions& read_options, ColumnFamilyHandle* column_family,
     const size_t num_keys, const Slice* keys, PinnableSlice* values,
     Status* statuses, bool sorted_input, ReadCallback* callback) {
+  const Comparator* const ucmp = rep->comparator.GetComparator(column_family);
```

Comment:
Question (not strictly related to this PR): does WBWI only support `MultiGet` with a single CF?

Reply:
Currently yes.